tt/console.html.tt

```diff
@@ -124,6 +135,11 @@ function xhr_request(url, method, data, callback, error_callback) {
 function update_content(response) {
     var refreshable = get_element('refreshable_content');
     refreshable.innerHTML = response;
+    var pages_top = document.getElementById('pages_top');
+    var incoming_pages = refreshable.getElementsByClassName('pages');
```

Comment:
Feels slightly hackish.
I suggest replacing `is_header` parameter of `pages.tti` with (optional) `id`